### PR TITLE
VM : support runcommand

### PR DESCRIPTION
**Pull request by yugangw-msft**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [x] Each command and parameter has a meaningful description.
- [x] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py [hunk 1]
```diff
@@ -955,6 +955,21 @@
           text: az vm assign-identity -g MyResourceGroup -n MyVm --role Reader
 """
 
+helps['vm run-command'] = """
+    type: group
+    short-summary: (PREVIEW) Manage run commands on a Virtual Machine
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py]: Check with @sptramer on the syntax for the preview label. His recent PR established a standard.

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py [hunk 2]
```diff
@@ -327,3 +337,7 @@ def get_vm_size_completion_list(prefix, action, parsed_args, **kwargs):  # pylin
     register_cli_argument(scope, 'duration_in_seconds', help='Time duration in seconds until the SAS access expires')
 
 register_cli_argument('vm format-secret', 'secrets', multi_ids_type, options_list=('--secrets', '-s'), help='Space separated list of Key Vault secret URIs. Perhaps, produced by \'az keyvault secret list-versions --vault-name vaultname -n cert1 --query "[?attributes.enabled].id" -o tsv\'')
+
+register_cli_argument('vm run-command invoke', 'parameters', nargs='+', help="space separated parameters in the format of 'name=value' or just 'value'")
+register_cli_argument('vm run-command invoke', 'scripts', nargs='+', help="script lines separated by whites spaces. Use @<file> to load from a file")
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py]: The most recent PR establishes `@{file}` as the standard way for denoting this.

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py [hunk 3]
```diff
@@ -327,3 +337,7 @@ def get_vm_size_completion_list(prefix, action, parsed_args, **kwargs):  # pylin
     register_cli_argument(scope, 'duration_in_seconds', help='Time duration in seconds until the SAS access expires')
 
 register_cli_argument('vm format-secret', 'secrets', multi_ids_type, options_list=('--secrets', '-s'), help='Space separated list of Key Vault secret URIs. Perhaps, produced by \'az keyvault secret list-versions --vault-name vaultname -n cert1 --query "[?attributes.enabled].id" -o tsv\'')
+
+register_cli_argument('vm run-command invoke', 'parameters', nargs='+', help="space separated parameters in the format of 'name=value' or just 'value'")
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py]: Recommend the same format as tags. `'key[=value]'`

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py (hunk 1 above)
Comment (yugangw-msft) [reply src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py]: Based on the [diff](https://github.com/Azure/azure-cli/pull/4346/files#diff-097ad97ce6bcf2e69f56c3456ff451b8) in @sptramer's PR, I believe I am conforming.

**Comment by derekbekoe** [comment]
@yugangw-msft there was no heading under `unreleased` in the HISTORY.rst file.
Can you add this in a separate PR?

**Comment by yugangw-msft** [comment]
@derekbekoe, i opened https://github.com/Azure/azure-cli/pull/4453

**Pull request merged**